### DISCUSSION TITLE
Number of documents openai

### DISCUSSION THREAD

**Discussion by DataGomes** [MaartenGr/BERTopic#1123] [Q&A]
When using the prompt with openai can I increase the number of documents used as the basis for the analysis, to decrease the bias that comes from the current number of documents. 

prompt = """
I have a topic that contains the following documents: 
[DOCUMENTS]
The topic is described by the following keywords: [KEYWORDS]

Based on the information above, extract a short topic label in the following format:
topic: <topic label>
"""

**Comment by MaartenGr** [comment]
I am indeed thinking of exposing parameters for increasing the number of keywords and documents but will have to think about a nice interface. Often, when you increase the number of documents, they remain relatively similar. Also exposing an option for including diverse documents would be helpful here.

**Reply by DataGomes** [reply]
Thank you for the answer! In general I agree that the documents tend to remain similar, in one case I found  that it created some bias and thought that increasing the number of documents, or the diversity of them as you suggested, could improve the results in general.
